aptible-cli.gemspec

```diff
@@ -28,6 +28,11 @@ Gem::Specification.new do |spec|
   spec.add_dependency 'term-ansicolor'
   spec.add_dependency 'chronic_duration', '~> 0.10.6'
 
+  if RUBY_PLATFORM =~ /mswin|mingw/i
```

Comment:
It looks like Rubygems looks for a selection of other patterns to achieve the same thing: https://github.com/rubygems/rubygems/blob/f68cdada86721d3adf4e8c106d277f99d2067cd6/lib/rubygems.rb#L122-L129 — do we have a particular reason to use a different subset?

(e.g. perhaps we *don't* actually want to use the Windows code on Cygwin?)

FWIW, if Rubygem's patterns work for us, perhaps we'd want to use `Gem.win_platform?` directly, but that's tangential.